fn_parse_autorate_log.m

```diff
@@ -787,6 +787,9 @@
 	log_struct.INFO = [];
 	log_struct.SHAPER = [];
 	log_struct.metainformation = [];
+  log_struct.SUMMARY_HEADER = [];
+	log_struct.SUMMAY = [];

```

Comment:
This looks like a typo, I could only see `log_struct.SUMMARY` being used.

Reply:
Indeed, good catch, thanks...